bool rede_neural_ligada